Themis/Views/Assessment/AssessmentView.swift

```diff
@@ -60,6 +61,9 @@ struct AssessmentView: View {
             .padding(.top, 4)
             .padding(.leading, 13)
         }
+        .onAppear {
+            ar.maxScore = maxScore
+        }
```

Comment:
I think it's a bit of a code smell that we have to do this, but I experimented a bit and didn't find a better solution, so I would leave it for now.

Reply:
Best solution is probably to do it in the initialiser of AssessmentView but then you would have to manually add a initialiser for all the other values as well which would probably not benefit the already giant AssessmentView